packages/ra-input-rich-text/package.json

```diff
@@ -45,6 +45,7 @@
         "react-dom": "^16.9.0"
     },
     "dependencies": {
+        "ra-ui-materialui": "^3.0.0",
```

Comment:
please move it to peerDependencies, just like `ra-core`

Reply:
done